test/test_server.ts

```diff
@@ -47,7 +47,7 @@ describe("the action hub", () => {
       .set("User-Agent", "LookerOutgoingWebhook/5.0.0")
       .end((_err, res) => {
         chai.expect(res).to.have.status(200)
```

Comment:
Rename the test as it is both Segment and Rudderstack now

Reply:
done